Makefile

```diff
@@ -23,13 +23,8 @@ STATICCHECK_IGNORE = \
 
 DOCKER_IMAGE_NAME       ?= prometheus
 
-ifdef DEBUG
-	bindata_flags = -debug
-endif
-
 .PHONY: assets
 assets:
 	@echo ">> writing assets"
-	@$(GO) get -u github.com/jteeuwen/go-bindata/...
-	@go-bindata $(bindata_flags) -pkg ui -o web/ui/bindata.go -ignore '(.*\.map|bootstrap\.js|bootstrap-theme\.css|bootstrap\.css)'  web/ui/templates/... web/ui/static/...
-	@$(GO) fmt ./web/ui
+	$(GO) get -u github.com/shurcooL/vfsgen
```

Comment:
Is this still needed given its vendored?

Reply:
Yes. I have removed it from `vendor/` in the last version because otherwise `govendor ilst +unused` complains that it isn't used.

Reply:
It has a `// +build ignore` constraint https://github.com/prometheus/prometheus/pull/4430/files#diff-eb481951f2846eb52fb4b11bfa5b8afdR14

Reply:
Using a blank import has made `govendor` happy.